### PR TITLE
Tests/jax components/executing/base

### LINKED ISSUE #530: [TEST] Jax Executing Base
### What do you want to test?
Jax executor init components

### Outline of test structure
* Unit tests
* Test components and hooks

### Definition of done
Passing checks, cover all hooks, edge cases considered

### Mandatory checklist before making a PR
* [ ] The success criteria laid down in “Definition of done” are met.
* [ ] Test code is documented - docstrings for methods and classes, static types for arguments.
* [ ] Documentation is updated - README, CONTRIBUTING, or other documentation.

### PR DESCRIPTION
**Pull request by OmaymaMahjoub**
## What?
Unit test for Jax executor init component
## Why?
Components are currently not tested. Needs to be robust for Jax Mava release
## How?
Test `ExecutorInit` component from `components/jax/executing/base.py`
## Extra
Close #530

### CODE REVIEW

**Review by RuanJohn — DISMISSED** [review]
Looks good. Thank you @OmaymaMahjoub!

File: tests/jax/components/executing/base_test.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+
+@pytest.fixture
+def mock_executor() -> Executor:
+    """Mock executor component.
+
+    Returns:
+        Executor
+    """
+    store = SimpleNamespace(is_evaluator=None, observations={})
```
Comment (AsadJeewa) [inline tests/jax/components/executing/base_test.py:63]: The empty dict observations can be removed from the store

**Review by AsadJeewa — DISMISSED** [review]
Thanks Omayma. Looks good

**Review by AsadJeewa — COMMENTED** [review]
beyond the small change to remove obs dict

File: tests/jax/components/executing/base_test.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    executor = Executor(store=store)
+    executor._interval = None
+    return executor
+
+
+# test on_building_init
```
Comment (AsadJeewa) [inline tests/jax/components/executing/base_test.py:69]: Also, please remove these comments above each test method as I do not think that they are needed

**Review by RuanJohn — APPROVED** [review]

**Review by AsadJeewa — APPROVED** [review]
